client/app/components/Resource/index.jsx

```diff
@@ -14,12 +14,17 @@ type Props = {
   updateTagFilter?: Function,
 };
 
+const scrollUp = () => {
+  window.scrollTo(0, 0);
+};
+
 const taggedResources = (
   tagged: ?boolean,
   tags: ?(string[]),
   updateTagFilter,
 ) => {
   if (tagged && tags) {
+    scrollUp();
```

Comment:
So this will work when you click tags, but when you click the "Load More" button at the bottom of the page, it will unintentionally scroll back up.

You'll only want to call this when the `Tag` component is clicked. You'll want to do this in the `onClick` listener.